google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1148,6 +1148,74 @@ def iter_crm_projects(self, parent_type, parent_id):
         for project in resources:
             yield project
 
+    # pylint: disable=inconsistent-return-statements, using-constant-test
+    def iter_crm_folder_org_policies(self, folder_id):
+        """Folder organization policy in a folder from Cloud Asset data.
+        Args:
+            folder_id (str): id of the folder to get policy.
+
+        Returns:
+            dict: Folder organization policy.
+        """
+        resource = self.dao.iter_cai_assets(
+            ContentTypes.org_policy,
+            'cloudresourcemanager.googleapis.com/Folder',
+            '//cloudresourcemanager.googleapis.com/{}'.format(folder_id),
+            self.engine)
+        if resource:
+            return resource
+
+    # pylint: disable=inconsistent-return-statements, using-constant-test
+    def iter_crm_organization_access_policies(self, org_id):
```

Comment:
Inside access policies, there are sub resources (e.g. service_perimeter and access_level) under access_policies, too. Just wanted to make sure those are also included.
Example access policy in the dump file:
```
{"name":"//cloudresourcemanager.googleapis.com/organizations/660570133860","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_policy":{"name":"accessPolicies/1008882730433","parent":"organizations/660570133860","title":"default policy"},"ancestors":["organizations/660570133860"]}
{"name":"//cloudresourcemanager.googleapis.com/organizations/660570133860","asset_type":"cloudresourcemanager.googleapis.com/Organization","service_perimeter":{"name":"accessPolicies/1008882730433/servicePerimeters/Test_Service_Parameter","title":"Test Service Parameter","status":{"resources":["projects/179891054368"],"access_levels":["accessPolicies/1008882730433/accessLevels/abcb"],"restricted_services":["container.googleapis.com"]}},"ancestors":["organizations/660570133860"]}
{"name":"//cloudresourcemanager.googleapis.com/organizations/660570133860","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_level":{"name":"accessPolicies/1008882730433/accessLevels/abcb","title":"abcb","basic":{"conditions":[{"ip_subnetworks":["0.0.0.0/0"],"regions":["US"]}]}},"ancestors":["organizations/660570133860"]}
{"name":"//cloudresourcemanager.googleapis.com/organizations/660570133860","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_level":{"name":"accessPolicies/1008882730433/accessLevels/Ebil","title":"Ebil","basic":{"conditions":[{"device_policy":{"require_screenlock":true,"allowed_encryption_statuses":[3]}}]}},"ancestors":["organizations/660570133860"]}
```

Reply:
Yes, it is included.

Reply:
Done